e2e/app.po.ts

```diff
@@ -103,7 +103,9 @@ export class AppPage {
 
   clickButton(buttonTitle: string): P<any> {
     log.info(`clicking button ${buttonTitle}`);
-    return this.getButton(buttonTitle).click();
+    const buttonElement = this.getButton(buttonTitle);
+    browser.wait(ExpectedConditions.presenceOf(buttonElement), 10000, 'Wait for button');
```

Comment:
Lets change this to `visibilityOf()` as it's a bit stronger. As it represents: "element is present in DOM && visible". Sorry for nit-picking here :)